ts/components/conversation/composition/CompositionButtons.tsx

```diff
@@ -50,7 +50,7 @@ export const StartRecordingButton = (props: { onClick: Noop }) => {
 };
 
 // eslint-disable-next-line react/display-name
-export const ToggleEmojiButton = React.forwardRef<HTMLDivElement, { onClick: Noop }>(
+export const ToggleEmojiButton = React.forwardRef<HTMLButtonElement, { onClick: Noop }>(
```

Comment:
if we get to React 19 one day we won't need forwardRef anymore as ref is included in all components 🙏

Reply:
well we will right, as part of onboarding?

Reply:
Sadly 19 isn't out yet. We do have 18 in onboarding though 👀